rust/src/lib.rs

```diff
@@ -888,7 +901,7 @@ pub unsafe extern "C" fn zcashlc_get_verified_transparent_balance_for_account(
             })
             .and_then(|target| {
                 db_data
-                    .get_transparent_receivers(account_uuid)
+                    .get_transparent_receivers(account_uuid, true)
```

Comment:
This could also be set to `false` but it should be harmless in the context of Zashi; we will never have queried for funds sent to these taddrs, so no funds sent to those taddrs should show up in the balance.